src/client/src/apps/MainRoute/widgets/spotTile/components/SpotTile.tsx

```diff
@@ -3,7 +3,7 @@ import { spotDateFormatter } from '../model/dateUtils'
 import NotionalInput from './notional'
 import PriceControls from './PriceControls'
 import TileHeader from './TileHeader'
-import { getDefaultInitialNotionalValue } from './Tile/TileBusinessLogic'
+import { getDefaultNotionalValue } from './Tile/TileBusinessLogic'
```

Comment:
a simple name change to remove redundant term (initial == default in this context?)